lib/strategies/base.js

```diff
@@ -67,8 +67,10 @@ BaseStrategy.prototype._attachListeners = function () {
     self._reformatInput(event);
   });
   self.inputElement.addEventListener('input', function (event) {
-    // Safari AutoFill fires CustomEvents -- mark the input as unformatted without actually unformatting the current value
-    if (event instanceof CustomEvent) {
+    // Safari AutoFill fires CustomEvents
```

Comment:
Gonna let the current build pass, but I'll update this comment to:


```js
// Safari AutoFill fires CustomEvents
// LastPass sends an `isTrusted: false` property
// Since the input is changed all at once, set isFormatted
// to false so that reformatting actually occurs
```